docs/dev.md

```diff
@@ -416,15 +420,33 @@ touch on the file being launched solves it.
 touch test/e2e/alerts.js; yarn test:e2e:alerts
 ```
 
+##### Client not registered or missing OAuth information
+
+When running twice the tests, the client oauth information is not correctly
+recovered from the JSON cache file resulting in
+
+```
+{ NotRegisteredException: Client not registered or missing OAuth information
+...
+```
+
+Removing the token fixes the problem
+
+```
+rm /tmp/cozy-client-oauth-cozy-tools:8080-banks.alerts-e2e.json
+```
```

Comment:
Maybe we can add a link to this documentation section when we catch a NotRegisteredException?

Reply:
I rm the file now so that this piece of doc is not needed.